core/lib/spree/core/preferences/preferable.rb

```diff
@@ -103,8 +103,8 @@ def clear_preferences
     preferences.keys.each { |pref| preferences.delete pref }
   end
 
-  def clear_preferences_for(preference_keys)
```

Comment:
This method was used only in the `restore_design_settings_to_defaults`, so I just replaced it